Consul + Vault - ACLs (#194)
hashicorp/katakoda: consul-secure-acl-vault/step4.md

### Create a role to map names in Vault to a Consul ACL policy

> In the previous step you created a policy, `consul-servers`, to define servers' permissions.

Reviewer: ```suggestion
In the previous step you created a policy, `consul-servers`, to define the server agents' permissions.
```

You are going to use that policy when creating tokens for Consul servers.

A recommended approach to generate tokens for identical policies is to associate the policy to a role.

> 

Reviewer: ```suggestion
```

> Roles allow for the grouping of a set of policies and service identities into a reusable higher-level entity that can be applied to many tokens.

Reviewer: ```suggestion
Roles allow you to group a set of policies and service identities into a reusable higher-level entity that can be applied to many tokens.
```
